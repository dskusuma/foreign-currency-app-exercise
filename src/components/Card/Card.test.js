import initStoryshots from './node_modules/@storybook/addon-storyshots';
initStoryshots();