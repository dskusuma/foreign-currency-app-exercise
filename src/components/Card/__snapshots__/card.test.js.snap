// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Card default 1`] = `
<div>
  <div
    className="currency-amount"
  >
    <div
      className="currency-amount__code"
    >
      IDR
    </div>
    <div
      className="currency-amount__converted-amount"
    >
      123000
    </div>
  </div>
  <div
    className="currency-name"
  >
    <div
      className="currency-name__code"
    >
      IDR
       - 
    </div>
    <div
      className="currency-name__name"
    >
      Indonesia Rupiah
    </div>
  </div>
  <div
    className="rate"
  >
    <div
      className="rate__amount"
    >
      1 
      USD
       = 
      IDR
       
      14000
    </div>
  </div>
</div>
`;

exports[`Storyshots Card filled 1`] = `
<div>
  <div
    className="currency-amount"
  >
    <div
      className="currency-amount__code"
    >
      EUR
    </div>
    <div
      className="currency-amount__converted-amount"
    >
      10000
    </div>
  </div>
  <div
    className="currency-name"
  >
    <div
      className="currency-name__code"
    >
      EUR
       - 
    </div>
    <div
      className="currency-name__name"
    >
      Euro
    </div>
  </div>
  <div
    className="rate"
  >
    <div
      className="rate__amount"
    >
      1 
      USD
       = 
      EUR
       
      1
    </div>
  </div>
</div>
`;
